### PR TITLE
Update control-access-based-on-network-location.md

### PR DESCRIPTION
**Pull request by SanjoyanM**
We have found a issue where users using Office.com was able to bypass this policy. Hence we have added the below line to make customers aware to configure the policy correctly for Office.com.

"For protecting access to SharePoint via Office.com portal we recommend using Azure Active Directory conditional access policy for SharePoint app and configure the trusted IP range there. "

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/control-access-based-on-network-location.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Here are some important considerations for setting a location-based policy:
...
     
     > [!NOTE]
-    > We recommend that when a location-based policy is enabled for SharePoint, the same policy and IP address ranges should be configured for Exchange and Yammer. SharePoint relies on these services to enforce that the users of these apps are within the trusted IP range.
+    > We recommend that when a location-based policy is enabled for SharePoint, the same policy and IP address ranges should be configured for Exchange and Yammer. SharePoint relies on these services to enforce that the users of these apps are within the trusted IP range. For protecting access to SharePoint via Office.com portal we recommend using Azure Active Directory conditional access policy for "Office 365" and configure the trusted IP range there.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/control-access-based-on-network-location.md:43]:
```suggestion
    > We recommend that when a location-based policy is enabled for SharePoint, the same policy and IP address ranges should be configured for Exchange and Yammer. SharePoint relies on these services to enforce that the users of these apps are within the trusted IP range. For protecting access to SharePoint via the Office.com portal we recommend using the Azure Active Directory conditional access policy for "Office 365" and configuring the trusted IP range there. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**